### PR TITLE
Hover state aren't paired with cursor

### PR DESCRIPTION
**Pull request by AnnapareddyHarsha**
## Current Behavior

Two  charts missing hover states
The percentage should not be present for numerical data
The light grey colour E5E5E5 doesn't pair with any semantic slot

## New Behavior

After this fix hover state is working as expected 
image
Fixed missing titles in hover sate for these: Grouped Vertical Bar Chart Styled
image

After this fix percentage is not present for numerical data

image

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/0a2e4c179791510becb1495d836a97e997f5f16d'>0a2e4c179791510becb1495d836a97e997f5f16d</a></sub>
<div id="bundleSizeComment9423"/>

File: packages/react-examples/src/react-charting/HorizontalBarChart/HorizontalBarChart.CustomCallout.Example.tsx [hunk 1]
```diff
@@ -27,7 +27,7 @@ export const HorizontalBarChartCustomCalloutExample: React.FunctionComponent<{}>
           horizontalBarChartdata: { x: 800, y: 15000 },
```
Comment (AtishayMsft) [inline packages/react-examples/src/react-charting/HorizontalBarChart/HorizontalBarChart.CustomCallout.Example.tsx:27]: All hover values are not fixed. one, seven, eight are still showing old value

File: packages/react-examples/src/react-charting/HorizontalBarChart/HorizontalBarChart.CustomCallout.Example.tsx [hunk 2]
```diff
@@ -39,7 +39,7 @@ export const HorizontalBarChartCustomCalloutExample: React.FunctionComponent<{}>
           horizontalBarChartdata: { x: 8888, y: 15000 },
           color: DefaultPalette.redDark,
```
Comment (AtishayMsft) [inline packages/react-examples/src/react-charting/HorizontalBarChart/HorizontalBarChart.CustomCallout.Example.tsx:40]: Also fix the hover values in HorizontalBarChart.CustomAccessibility example

Comment (AnnapareddyHarsha) [reply packages/react-examples/src/react-charting/HorizontalBarChart/HorizontalBarChart.CustomCallout.Example.tsx:40]: Changes are done accordingly

File: packages/react-examples/src/react-charting/HorizontalBarChart/HorizontalBarChart.CustomCallout.Example.tsx (hunk 1 above)
Comment (AnnapareddyHarsha) [reply packages/react-examples/src/react-charting/HorizontalBarChart/HorizontalBarChart.CustomCallout.Example.tsx:27]: Fixed accordingly

**Review by AtishayMsft — APPROVED** [review]
This is not fixed

**Pull request merged**